### PR TITLE
Changes While Loop To For Loop

### PR DESCRIPTION
**Pull request by Tails2091**
Why was this a while loop?
Seriously if someone has an answer, please.
:cl:
fix: Changed wannabe for loop to a real for loop.
/:cl:

### CODE REVIEW

File: code/game/objects/items/random_items.dm [hunk 1]
```diff
@@ -130,8 +130,7 @@
 
 /obj/item/storage/pill_bottle/random_meds/New()
 	..()
-	var/i = 1
-	while(i < storage_slots)
+	for(var/i=1, i<storage_slots, i++)						// Why was this a while loop?
```
Comment (Fox-McCloud) [inline code/game/objects/items/random_items.dm]: use the proper format, please.

Comment (Tails2091) [reply code/game/objects/items/random_items.dm]: what do you mean by proper format?

Comment (Fox-McCloud) [reply code/game/objects/items/random_items.dm]: `for(var/i in 1 to X)`

Comment (KasparoVy) [reply code/game/objects/items/random_items.dm]: [This contributing guide](https://github.com/ParadiseSS13/Paradise/blob/master/.github/CONTRIBUTING.md) has a helpful section describing the 'most properer' format and why it's used. BYOND seems to favour it over all other styles of for loop

**Pull request merged**